lib/src/theme/default_theme.dart

```diff
@@ -167,4 +182,19 @@ class DarkChatTheme extends ChatTheme {
 
   @override
   String? get htmlTextFontFamily => 'Avenir';
+
+  @override
+  // TODO: implement chatIcon
+  Widget get chatIcon => const FlutterLogo();
+
+  @override
+  // TODO: implement readMoreMessageLength
+  double get readMoreMessageLength => 200;
+
+  @override
+  // TODO: implement sendMessageIcon
+  Widget get sendMessageIcon => const Icon(
```

Comment:
Let's parameterize the whole sendMessage icon then (including it's background etc.)